config/warp_tokens.ts

```diff
@@ -5,6 +5,7 @@ import type { WarpRouteConfig } from '../src/warp/config';
 // A config for deploying Warp Routes to a set of chains
 // Not required for Hyperlane core deployments
 export const warpRouteConfig: WarpRouteConfig = {
+  // votable:true  You can enable this config if you want to make your synthetic ERC20 tokens votable
```

Comment:
Put these on different lines to make it obvious that votable is something that can be commented out

Reply:
Actually, shouldn't this be on the synthetics part of the config? So move it to line 21?

Reply:
I have updated it. Now you have the votable options both in the base as well as the synthetic chain. If votable is true in the base chain it will deploy the HypERC20CollateralVotable.sol and similarly, in the synthetic chain HypERC20Votable.sol